pkg/ip/ip_test.go

```diff
@@ -61,10 +62,22 @@ func (s *IPTestSuite) testIPNetsEqual(created, expected []*net.IPNet, c *C) {
 	}
 }
 
+func (s *IPTestSuite) testIPsEqual(created, expected net.IP, c *C) {
+	for k := range created {
+		c.Assert(reflect.DeepEqual(created[k], expected[k]), Equals, true)
```

Comment:
I wonder if we should standardize using `comparator.DeepEquals()` rather than `reflect.DeepEqual`, so if anything like this ever fails then we get nice diff'd output. Maybe this PR just predates our comparator pkg.

Reply:
I'm going to move these tests to use `c.Assert(x, Equals, y)` as these are all primitive types and don't need `DeepEquals`. Will have that done in next PR.